### ISSUE TITLE
Slots available but not able to book, or proceed with booking

### ISSUE THREAD

**Issue by pramonty** [cowinapi/developer.cowin#254]
https://user-images.githubusercontent.com/55252165/118496178-83fbb300-b741-11eb-9bff-bb669818f94c.mp4

**Comment by pramonty** [comment]
Could this be because in the json response, available_capacity_dose1 is 150, whereas available_capacity is 0..??

**Comment by Abhishek150995** [comment]
Are you logged in before trying to proceed with booking?

**Comment by mjaiswal1905** [comment]
May be you have some addon which prevents JavaScript execution or something similar? Could try to do it in incognito mode?

**Comment by pramonty** [comment]
@Abhishek150995 yeah, I was logged in. And the fact that it has been 4 hours and not a single of those slots have been booked reinforces my belief that it's an issue at the cowin site...

**Comment by pramonty** [comment]
@mjaiswal1905 I was able to book slots for 45+ at a different center. And the fact that it has been 4 hours and not a single of those slots have been booked in this cutthroat competition reinforces my belief that it's an issue at the cowin site. Maybe how the open slots were uploaded...?? When I inspect the JSON response for those sessions, I see available_capacity to be 0, but available_capacity_dose1 to be 150...so perhaps java script has a trigger only for the sessions having available_capacity>0

**Comment by pramonty** [comment]
@cowinapi hope this helps. I see this for district id 457 and session ids: 2d381621-f1ba-4d32-a6d3-a9ce53717027, b921d813-a0ac-4900-a2c6-ad9dcca0134e, 4aabe361-7a64-4da6-ad98-65e8038d42af, 8265142d-0094-461b-9c12-a0957830e21a, f3a31c59-1b4d-43d4-917f-7f7fe8c5078a and 8c416a87-93e1-46be-a64f-c9cc059310a9
